src/engine/RuntimeInformation.h

```diff
@@ -39,12 +39,14 @@ class RuntimeInformation {
 
   /// The total time spent computing this operation. This includes the
   /// computation of the children.
-  double totalTime_ = 0.0;
+  std::chrono::milliseconds totalTime_ = std::chrono::milliseconds::zero();
```

Comment:
0ms

Reply:
Unfortunately we can't use `using namespace` within a class without cluttering the global namespace. I went with an alternative that is slightly better than this